packages/vapor/scss/colors.scss

```diff
@@ -43,19 +162,46 @@ $colors: (
     'deprecated-medium-blue': #1d4f76,
     'deprecated-dark-blue': #263e55,
     'deprecated-oxford-blue': #313a45,
-    'salesforce-blue': #009ddc,
-    'white': #ffffff,
-    'black': #000000,
 );
 
-@function hexToRGB($hex) {
-    @return red($hex), green($hex), blue($hex);
+@function toRGB($color) {
+    @return red($color), green($color), blue($color);
 }
 
-/* css variables*/
-:root {
-    @each $name, $hexValue in $colors {
-        --#{$name}: #{$hexValue};
-        --#{$name}-rgb: #{hexToRGB($hexValue)};
+@mixin spreadPalette($palette) {
+    @each $name, $color in $palette {
+        --#{$name}: #{$color};
+        --#{$name}-rgb: #{toRGB($color)};
     }
 }
+
+:root {
+    @include spreadPalette($primary);
+    @include spreadPalette($secondary);
+    @include spreadPalette($semantic);
+    @include spreadPalette($partners);
+    @include spreadPalette($deprecated);
+
+    // semantic derivates
+    --info-10: var(--navy-blue-10);
+    --info-10-rgb: var(--navy-blue-10-rgb);
+    --info-60: var(--navy-blue-60);
+    --info-60-rgb: var(--navy-blue-60-rgb);
+    --info-100: var(--navy-blue-100);
+    --info-100-rgb: var(--navy-blue-100-rgb);
+    --warning-10: var(--turbo-yellow-10);
+    --warning-10-rgb: var(--turbo-yellow-10-rgb);
+    --warning-70: var(--turbo-yellow-70);
+    --warning-70-rgb: var(--turbo-yellow-70-rgb);
+    --warning-90: var(--turbo-yellow-90);
+    --warning-90-rgb: var(--turbo-yellow-90-rgb);
+    --critical-10: var(--pomegranate-10);
+    --critical-10-rgb: var(--pomegranate-10-rgb);
+    --critical-70: var(--pomegranate-70);
+    --critical-70-rgb: var(--pomegranate-70-rgb);
```

Comment:
I see you've removed variables that aren't divisible by 20 but you've added them in here (10, 70, 90), is there a reason for this?

Reply:
I just basically removed colors names that didn't exist in figma and added the ones that we're not there. No dark magic math here hehe